components/nav.js

```diff
@@ -1,12 +1,14 @@
 import Link from 'next/link';
 import Image from 'next/image';
+import { useRouter } from 'next/dist/client/router';
 import { Fragment } from 'react';
 import { Popover, Disclosure, Transition } from '@headlessui/react';
 import { MenuIcon, XIcon } from '@heroicons/react/outline';
 import { ChevronDownIcon } from '@heroicons/react/solid';
-
+import { Link as ScrollLink } from 'react-scroll';
 import { get_waardeketens, get_over } from '../utils/nav_structure';
 import CirculawLogo from '../public/Circulaw_logotype.png';
+import logo from '../public/Circulaw_logotype_home.png';
```

Comment:
we have do better naming for different logos here to reduce confusion,  but just a minor thing